Java/Complex/Complex.java

```diff
@@ -0,0 +1,61 @@
+import java.util.Vector;
+
+/**
+ * Created by andrew on 18.11.16.
+ */
+public class Complex implements Cloneable{
+    private double real;
+    private double imaginary;
+
+    public Complex(double real, double imaginary) {
+        this.real = real;
+        this.imaginary = imaginary;
+    }
+
+    public Complex() {
+        this(0.0, 0.0);
+    }
+
+    public double getReal() {
+        return real;
+    }
+
+    public double getImaginary() {
+        return imaginary;
+    }
+
+    public void setReal(double real) {
+        this.real = real;
+    }
+
+    public void setImaginary(double imaginary) {
+        this.imaginary = imaginary;
+    }
+
+    public Complex sum(Complex other) {
+        return new Complex(real + other.real, imaginary + other.imaginary);
+    }
+
+    public Complex diff(Complex other) {
+        return new Complex(real - other.real, imaginary - other.imaginary);
+    }
+
+    public Complex mult(Complex other) {
+        return new Complex(real * other.real - imaginary * other.imaginary, real * other.imaginary + imaginary * other.real);
```

Comment:
Do not write more than 120 symbols in one line. Line that is longer than 120 symbols is difficult to read - need to scroll horizontal.
In IDEA you can set mark line which  will show you the limit for 120 symbols line.
Read about [wrapping line](http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-136091.html#248)
Try to make your code readable